### DISCUSSION TITLE
Give up Host and Mining, All in Tapp Token to make you be rich?

### DISCUSSION THREAD

**Discussion by nick-yin** [tearust/teaproject#104] [General]
Judging from this season, you can actually give up doing HOST and mining. The income is too small or even negligible. You can make a lot of money by all in TAPP Token. I don't think this is good for TEA Project, The benefits of each role in TEA Project should be relatively balanced.

**Comment by kevingzhang** [comment]
In TEA project community, there are different kinds of community members join for different purpose. 
some are miners, some are investors, some are developers. most may have one or more roles together.
This is adjust automatically by the market.
If more people give up mining but to invest to the TApp, the few hosts would be available to support. The pricing of host would be higher since the TApp owners would have to compete with other tapp to gain the support of miners. On the other hand, if there are too many hosts, the tapp owners do not have to compete with other tapp to win the hosts the price of host will be dropped.

Right now you can see we are at the host > tapp circle. When most CML die, you will see those investors or TApp owners lost their profit since they have to pay more for hosting.
